From 58d76eaf6684c56f07bdbf6a57d433526e823293 Mon Sep 17 00:00:00 2001
From: "Ch1llb0/skin.osmc" <36623079+Ch1llb0@users.noreply.github.com>
Date: Wed, 19 Dec 2018 11:03:08 +0100
Subject: [PATCH] Update debug overlay

Custom_Overlay_Debug.xml:
- add viewmode, sortmode and sortorder labels to window labels
- add new window xmls
- update old window xmls
- remove deprecated window xmls
---
 16x9/Custom_Overlay_Debug.xml | 301 ++++++++--------------------------
 1 file changed, 68 insertions(+), 233 deletions(-)

diff --git a/16x9/Custom_Overlay_Debug.xml b/16x9/Custom_Overlay_Debug.xml
index e423db8..bc83a40 100755
--- a/16x9/Custom_Overlay_Debug.xml
+++ b/16x9/Custom_Overlay_Debug.xml
@@ -32,7 +32,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>AddonBrowser.xml</label>
+				<label>AddonBrowser.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(AddonBrowser.xml)</visible>
 			</control>
 
@@ -43,7 +43,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>FileManager.xml</label>
+				<label>FileManager.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(FileManager.xml)</visible>
 			</control>
 
@@ -54,7 +54,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>Home.xml</label>
+				<label>Home.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(Home.xml)</visible>
 			</control>
 
@@ -65,7 +65,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>LoginScreen.xml</label>
+				<label>LoginScreen.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(LoginScreen.xml)</visible>
 			</control>
 
@@ -76,7 +76,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyMusicNav.xml</label>
+				<label>MyMusicNav.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(MyMusicNav.xml)</visible>
 			</control>
 
@@ -87,7 +87,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyMusicPlaylist.xml</label>
+				<label>MyMusicPlaylist.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(MyMusicPlaylist.xml)</visible>
 			</control>
 
@@ -98,7 +98,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyMusicPlaylistEditor.xml</label>
+				<label>MyMusicPlaylistEditor.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(MyMusicPlaylistEditor.xml)</visible>
 			</control>
 
@@ -109,10 +109,10 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyMusicSongs.xml</label>
-				<visible>Window.IsVisible(MyMusicSongs.xml)</visible>
+				<label>MyPics.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
+				<visible>Window.IsVisible(MyPics.xml)</visible>
 			</control>
-
+			
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -120,8 +120,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyPics.xml</label>
-				<visible>Window.IsVisible(MyPics.xml)</visible>
+				<label>MyPlaylist.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
+				<visible>Window.IsVisible(MyPlaylist.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -131,7 +131,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyPrograms.xml</label>
+				<label>MyPrograms.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(MyPrograms.xml)</visible>
 			</control>
 
@@ -142,7 +142,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyPVRChannels.xml</label>
+				<label>MyPVRChannels.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(MyPVRChannels.xml)</visible>
 			</control>
 
@@ -153,7 +153,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyPVRGuide.xml</label>
+				<label>MyPVRGuide.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(MyPVRGuide.xml)</visible>
 			</control>
 
@@ -164,7 +164,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyPVRRecordings.xml</label>
+				<label>MyPVRRecordings.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(MyPVRRecordings.xml)</visible>
 			</control>
 
@@ -175,7 +175,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyPVRSearch.xml</label>
+				<label>MyPVRSearch.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(MyPVRSearch.xml)</visible>
 			</control>
 
@@ -186,7 +186,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyPVRTimers.xml</label>
+				<label>MyPVRTimers.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(MyPVRTimers.xml)</visible>
 			</control>
 
@@ -197,7 +197,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyVideoNav.xml</label>
+				<label>MyVideoNav.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(MyVideoNav.xml)</visible>
 			</control>
 
@@ -208,8 +208,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyVideoPlaylist.xml</label>
-				<visible>Window.IsVisible(MyVideoPlaylist.xml)</visible>
+				<label>MyWeather.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
+				<visible>Window.IsVisible(MyWeather.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -219,8 +219,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MyWeather.xml</label>
-				<visible>Window.IsVisible(MyWeather.xml)</visible>
+				<label>Settings.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
+				<visible>Window.IsVisible(Settings.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -230,8 +230,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>Settings.xml</label>
-				<visible>Window.IsVisible(Settings.xml)</visible>
+				<label>SettingsCategory.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
+				<visible>Window.IsVisible(SettingsCategory.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -241,10 +241,10 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>SettingsCategory.xml</label>
-				<visible>Window.IsVisible(SettingsCategory.xml)</visible>
+				<label>SettingsProfile.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
+				<visible>Window.IsVisible(SettingsProfile.xml)</visible>
 			</control>
-
+			
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -252,8 +252,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>SettingsProfile.xml</label>
-				<visible>Window.IsVisible(SettingsProfile.xml)</visible>
+				<label>SettingsScreenCalibration.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
+				<visible>Window.IsVisible(SettingsScreenCalibration.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -263,7 +263,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>SkinSettings.xml</label>
+				<label>SkinSettings.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(SkinSettings.xml)</visible>
 			</control>
 
@@ -274,7 +274,7 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>VideoFullScreen.xml</label>
+				<label>VideoFullScreen.xml $INFO[Container.Viewmode,(,]$INFO[Container.SortMethod, - ,]$INFO[Container.SortOrder, - ,)]</label>
 				<visible>Window.IsVisible(VideoFullScreen.xml)</visible>
 			</control>
 
@@ -302,17 +302,6 @@
 				<visible>Window.IsVisible(DialogAddonSettings.xml)</visible>
 			</control>
 
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>DialogAlbumInfo.xml</label>
-				<visible>Window.IsVisible(DialogAlbumInfo.xml)</visible>
-			</control>
-
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -334,7 +323,7 @@
 				<label>DialogButtonMenu.xml</label>
 				<visible>Window.IsVisible(DialogButtonMenu.xml)</visible>
 			</control>
-
+			
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -342,8 +331,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogContentSettings.xml</label>
-				<visible>Window.IsVisible(DialogContentSettings.xml)</visible>
+				<label>DialogConfirm.xml</label>
+				<visible>Window.IsVisible(DialogConfirm.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -379,17 +368,6 @@
 				<visible>Window.IsVisible(DialogFavourites.xml)</visible>
 			</control>
 
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>DialogFileStacking.xml</label>
-				<visible>Window.IsVisible(DialogFileStacking.xml)</visible>
-			</control>
-
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -408,32 +386,10 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogKaiToast.xml</label>
-				<visible>Window.IsVisible(DialogKaiToast.xml)</visible>
+				<label>DialogGameControllers.xml</label>
+				<visible>Window.IsVisible(DialogGameControllers.xml)</visible>
 			</control>
-
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>DialogKaraokeSongSelector.xml</label>
-				<visible>Window.IsVisible(DialogKaraokeSongSelector.xml)</visible>
-			</control>
-
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>DialogKaraokeSongSelectorLarge.xml</label>
-				<visible>Window.IsVisible(DialogKaraokeSongSelectorLarge.xml)</visible>
-			</control>
-
+			
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -445,17 +401,6 @@
 				<visible>Window.IsVisible(DialogKeyboard.xml)</visible>
 			</control>
 
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>DialogMediaFilter.xml</label>
-				<visible>Window.IsVisible(DialogMediaFilter.xml)</visible>
-			</control>
-
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -474,21 +419,10 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogNetworkSetup.xml</label>
-				<visible>Window.IsVisible(DialogNetworkSetup.xml)</visible>
-			</control>
-
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>DialogNumeric.xml</label>
-				<visible>Window.IsVisible(DialogNumeric.xml)</visible>
+				<label>DialogMusicInfo.xml</label>
+				<visible>Window.IsVisible(DialogMusicInfo.xml)</visible>
 			</control>
-
+			
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -496,8 +430,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogOK.xml</label>
-				<visible>Window.IsVisible(DialogOK.xml)</visible>
+				<label>DialogNotification.xml</label>
+				<visible>Window.IsVisible(DialogNotification.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -507,8 +441,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogPeripheralManager.xml</label>
-				<visible>Window.IsVisible(DialogPeripheralManager.xml)</visible>
+				<label>DialogNumeric.xml</label>
+				<visible>Window.IsVisible(DialogNumeric.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -518,8 +452,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogPeripheralSettings.xml</label>
-				<visible>Window.IsVisible(DialogPeripheralSettings.xml)</visible>
+				<label>DialogPictureInfo.xml</label>
+				<visible>Window.IsVisible(DialogPictureInfo.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -529,8 +463,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogPictureInfo.xml</label>
-				<visible>Window.IsVisible(DialogPictureInfo.xml)</visible>
+				<label>DialogPlayerProcessInfo.xml</label>
+				<visible>Window.IsVisible(DialogPlayerProcessInfo.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -540,10 +474,10 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogProgress.xml</label>
-				<visible>Window.IsVisible(DialogProgress.xml)</visible>
+				<label>DialogPVRChannelGuide.xml</label>
+				<visible>Window.IsVisible(DialogPVRChannelGuide.xml)</visible>
 			</control>
-
+			
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -566,39 +500,6 @@
 				<visible>Window.IsVisible(DialogPVRChannelsOSD.xml)</visible>
 			</control>
 
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>DialogPVRGroupManager.xml</label>
-				<visible>Window.IsVisible(DialogPVRGroupManager.xml)</visible>
-			</control>
-
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>DialogPVRGuideInfo.xml</label>
-				<visible>Window.IsVisible(DialogPVRGuideInfo.xml)</visible>
-			</control>
-
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>DialogPVRGuideOSD.xml</label>
-				<visible>Window.IsVisible(DialogPVRGuideOSD.xml)</visible>
-			</control>
-
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -617,19 +518,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogPVRRecordingInfo.xml</label>
-				<visible>Window.IsVisible(DialogPVRRecordingInfo.xml)</visible>
-			</control>
-
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>DialogPVRTimerSettings.xml</label>
-				<visible>Window.IsVisible(DialogPVRTimerSettings.xml)</visible>
+				<label>DialogPVRInfo.xml</label>
+				<visible>Window.IsVisible(DialogPVRInfo.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -653,7 +543,7 @@
 				<label>DialogSelect.xml</label>
 				<visible>Window.IsVisible(DialogSelect.xml)</visible>
 			</control>
-
+			
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -661,8 +551,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogSlider.xml</label>
-				<visible>Window.IsVisible(DialogSlider.xml)</visible>
+				<label>DialogSettings.xml</label>
+				<visible>Window.IsVisible(DialogSettings.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -672,8 +562,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogSongInfo.xml</label>
-				<visible>Window.IsVisible(DialogSongInfo.xml)</visible>
+				<label>DialogSlider.xml</label>
+				<visible>Window.IsVisible(DialogSlider.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -697,7 +587,7 @@
 				<label>DialogTextViewer.xml</label>
 				<visible>Window.IsVisible(DialogTextViewer.xml)</visible>
 			</control>
-
+			
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -727,8 +617,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>DialogYesNo.xml</label>
-				<visible>Window.IsVisible(DialogYesNo.xml)</visible>
+				<label>EventLog.xml</label>
+				<visible>Window.IsVisible(EventLog.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -749,19 +639,8 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>LockSettings.xml</label>
-				<visible>Window.IsVisible(LockSettings.xml)</visible>
-			</control>
-
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>MusicKaraokeLyrics.xml</label>
-				<visible>Window.IsVisible(MusicKaraokeLyrics.xml)</visible>
+				<label>GameOSD.xml</label>
+				<visible>Window.IsVisible(GameOSD.xml)</visible>
 			</control>
 
 			<control type="label">
@@ -782,10 +661,10 @@
 				<shadowcolor>FF000000</shadowcolor>
 				<font>font13</font>
 				<align>left</align>
-				<label>MusicOverlay.xml</label>
-				<visible>Window.IsVisible(MusicOverlay.xml)</visible>
+				<label>MusicVisualisation.xml</label>
+				<visible>Window.IsVisible(MusicVisualisation.xml)</visible>
 			</control>
-
+			
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -808,39 +687,6 @@
 				<visible>Window.IsVisible(PlayerControls.xml)</visible>
 			</control>
 
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>ProfileSettings.xml</label>
-				<visible>Window.IsVisible(ProfileSettings.xml)</visible>
-			</control>
-
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>SettingsScreenCalibration.xml</label>
-				<visible>Window.IsVisible(SettingsScreenCalibration.xml)</visible>
-			</control>
-
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>SettingsSystemInfo.xml</label>
-				<visible>Window.IsVisible(SettingsSystemInfo.xml)</visible>
-			</control>
-
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -907,17 +753,6 @@
 				<visible>Window.IsVisible(VideoOSDSettings.xml)</visible>
 			</control>
 
-			<control type="label">
-				<width>780</width>
-				<height>21</height>
-				<textcolor>FFFF0000</textcolor>
-				<shadowcolor>FF000000</shadowcolor>
-				<font>font13</font>
-				<align>left</align>
-				<label>VideoOverlay.xml</label>
-				<visible>Window.IsVisible(VideoOverlay.xml)</visible>
-			</control>
-
 			<control type="label">
 				<width>780</width>
 				<height>21</height>
@@ -941,7 +776,7 @@
 			<shadowcolor>FF000000</shadowcolor>
 			<font>font13</font>
 			<align>right</align>
-			<label>$INFO[System.CurrentControl] - $INFO[System.CurrentControlID]</label>
+			<label>$INFO[System.CurrentControl]$INFO[System.CurrentControlID, - ,]</label>
 			<visible>Skin.HasSetting(DebugInfo)</visible>
 		</control>
 
